packages/client/src/pages/create-card/CreateCard.tsx

```diff
@@ -1,13 +1,17 @@
 import { z } from 'zod';
 
+import ChangeAvatar from '../../components/avatar/ChangeAvatar';
 import Button from '../../components/button/Button';
 import Form from '../../components/form/Form';
 import { useForm } from '../../components/form/hook/useForm';
 import Input from '../../components/form/input/Input';
+import TextArea from '../../components/form/text-area/TextArea';
 
 const createCardSchema = z.object({
   firstName: z.string().min(1, 'First Name must be at least 1 characters long!'),
   lastName: z.string().min(1, 'Last Name must be at least 1 characters long!'),
+  email: z.string().email(),
+  textarea: z.string().optional(),
```

Comment:
`textarea` property here does not tell me what is represented as textarea input(is it bio, is it some note ???) ?

Reply:
Also, I  would like to have an avatar in my validation schema.